maven-jxr/pom.xml

```diff
@@ -92,7 +92,11 @@ under the License.
       <groupId>org.codehaus.plexus</groupId>
       <artifactId>plexus-utils</artifactId>
     </dependency>
-
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.30</version>
```

Comment:
better would be to use the latest available with 3.1.1 -  to avoid any linking exceptions